Lesson33/Gokhan/src/components/UserDetailsForm/index.jsx

```diff
@@ -0,0 +1,33 @@
+import './styles.css';
+import { useContext, useState } from 'react';
+import { UserContex } from '../../context/UserContext';
+
+export const UserDetailsForm = ({ username, onUsernameChange }) => {
```

Comment:
Remove all props since the goal of this assignment is to manage user data through context only